app/src/main/java/com/example/kotlinplannerapp/home/HomeActivity.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.kotlinplannerapp.home
+
+class HomeActivity {
```

Comment:
[CDC] Tiene que extender de AppCompatActivity y sobreescribir el onCreate
```suggestion
class HomeActivity: AppCompatActivity() {
      ...
}
```